scripts/core/menu/notifications/notifications.js

```diff
@@ -66,6 +68,7 @@ function UserNotificationsService(
 
                     item._unread = !isRead(recipients, identity._id, true);
                     this.unread += item._unread ? 1 : 0;
+                    item.message = gettext(item.message, item.data);
```

Comment:
@ioanpocol you might need this change for the `ActivityMessageService` as well.  see https://github.com/superdesk/superdesk-client-core/blob/master/scripts/apps/stream/services/index.ts#L10